pkgs/tools/networking/ht-rust/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "ht-rs-${version}";
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "ducaale";
+    repo = "ht";
+    rev = "v${version}";
+    sha256 = "jT4Y6Yr3Cs2YPDSPNxTdIbyl6dP8v+HqG2zQJnPHvHA=";
+  };
+
+  cargoSha256 = "sha256-GTH7rXbCWLl/R1xUdA5dYgx592rw5ITQuqDoNoXCL2g=";
+
+  cargoPatches = [
+    # a patch file to change package & output binary name as "ht" conflicts with another package
+    ./bin-rename.patch
```

Comment:
Is it really necessary to patch the source?  Can you just move the resulting executable to a new name like this?
```nix
{
  postInstall = ''
    mv $out/bin/ht $out/bin/ht-rs
  '';
}
```